doc/source/dev/contributor/rendering_documentation.rst

```diff
@@ -188,6 +163,71 @@ preamble similar to the one below, indicating that this is an executable file:
 
 You don't need to edit this preamble, as it is autogenerated.
 
+Links, formatting and images
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When converting a notebook to a MyST Markdown file, you may need to adjust some
+of the content to match the MyST Markdown syntax.
+
+- **External links:** MyST Markdown uses the standard Markdown syntax for links.
+  For example, ``[link text](https://www.example.com)``.
+- **Internal links:** For internal cross-references such as links to SciPy
+  classes or functions, as well as intersphinx links, you can use the following
+  syntax: ``{role}`link text <reference>```, where ``role`` can be ``ref``,
+  ``class``, ``func`` or any other role you would use with reST. For example, to
+  link to the ``scipy.stats.bartlett`` function, use
+  ``{func}`scipy.stats.bartlett```.
+- **Formatting:** MyST Markdown supports standard Markdown formatting, such as
+  bold text, italic text, and code blocks. For example, to make text bold, you
+  can use double asterisks: ``**bold text**``. To make text monospace/formatted
+  as code, you can use *single* backticks: ```code```.
+- **Images:** If your notebook contains static images, you can include them in
+  the MyST Markdown file by using the following syntax:
+  ``![alt text](path/to/image.png)``. Images are usually stored in the same
+  folder as the MyST Markdown file, but you can also use relative paths to
+  reference images in other folders. Note that outputs of executed cells should
+  not be included in version control, as they will be automatically generated
+  when the notebook is executed during the SciPy documentation build.
+- **Linking to the MyST markdown pages:** MyST markdown supports adding link
+  anchors, which can be used to link to specific pages or document sections from
+  other documents. To add an anchor to a page, add ``(anchor_name)=`` to the
+  desired place in the document to be referenced. From other markdown pages, you
+  can link to this anchor using the following syntax: ``{ref}`anchor_name```.
+  From other reST pages, you can link to this anchor using the following syntax:
+  ``:ref:`anchor_name```.
+
+Making the notebook interactive
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you want to enable interactivity for your notebook when it is rendered in
+the SciPy documentation website, you need to add the following snippet at the
+beginning of the markdown file:
+
+.. code-block:: markdown
+
+    ```{eval-rst}
+    .. notebooklite:: <filename>.md
+       :new_tab: True
+    ```
```

Comment:
Here, I think we should 1. link to the `NotebookLite` directive's documentation (in case users want to use other options), and 2. also add a link to the `JupyterLite` directive's documentation (even if we don't recommend using it).

Reply:
The JupyterLite-sphinx docs are not that large, I think adding a link to the notebooklite directive docs is enough to point folks in the right direction. Wdyt?

Reply:
Sounds good!